public/src/main/java/com/revenuecat/purchases/models/Period.kt

```diff
@@ -0,0 +1,71 @@
+package com.revenuecat.purchases.models
+
+import android.os.Parcelable
+import kotlinx.parcelize.Parcelize
+
+/**
+ * Represents subscription or pricing phase billing period
+ */
+@Parcelize
+data class Period(
+    /**
+     * The number of period units.
+     */
+    val value: Int,
+
+    /**
+     * The increment of time that a subscription period is specified in.
+     */
+    val unit: Unit,
+
+    /**
+     * Specified in ISO 8601 format. For example, P1W equates to one week,
+     * P1M equates to one month, P3M equates to three months, P6M equates to six months,
+     * and P1Y equates to one year
+     */
+    val iso8601: String
+) : Parcelable {
+    @SuppressWarnings("MagicNumber")
+    enum class Unit(val identifier: Int) {
```

Comment:
does the identifier do anything for us? i don't think we have to have one...

Reply:
Oh... I don't think so 😅 Deleting!